src/cfn/terminalStackStates.ts

```diff
@@ -0,0 +1,28 @@
+// http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-listing-event-history.html
+// CREATE_COMPLETE | CREATE_FAILED | CREATE_IN_PROGRESS |
+// DELETE_COMPLETE | DELETE_FAILED | DELETE_IN_PROGRESS |
+// DELETE_SKIPPED | UPDATE_COMPLETexport E | UPDATE_FAILED |
```

Comment:
Lots more of these `exports` in here 😬

Reply:
There’s a bug in the ts refactoring tool version I started on. I’m going to rebase:rewrite them outfits the history.